gravitee-node-cache/gravitee-node-cache-plugin-redis/src/main/java/io/gravitee/node/plugin/cache/redis/configuration/RedisOptionsFactory.java

```diff
@@ -71,8 +75,13 @@ private static RedisOptions buildRedisOptions(RedisConfiguration configuration)
             VertxTcpClientFactory.configureSslClientOption(options.getNetClientOptions(), sslConfiguration);
         }
 
-        // Set max waiting handlers high enough to manage high throughput since we are not using the pooled mode
-        options.setMaxWaitingHandlers(1024);
+        options.setPoolName("cache-redis-" + UUID.randomUUID());
+        ofNullable(configuration.getMaxPoolSize()).ifPresent(options::setMaxPoolSize);
+        ofNullable(configuration.getMaxPoolWaiting()).ifPresent(options::setMaxPoolWaiting);
+        ofNullable(configuration.getPoolCleanerInterval()).ifPresent(options::setPoolCleanerInterval);
+        ofNullable(configuration.getPoolRecycleTimeout()).ifPresent(options::setPoolRecycleTimeout);
+        ofNullable(configuration.getMaxWaitingHandlers()).ifPresent(options::setMaxWaitingHandlers);
+
```

Comment:
Shouldn't we set some default value for all of those properties ?

Reply:
RedisOption already provide default values, that's why I didn't provide ones and rely on redis.